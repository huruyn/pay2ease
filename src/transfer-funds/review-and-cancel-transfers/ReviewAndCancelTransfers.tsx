import { Col, Row, Table } from 'react-bootstrap';
import CommonPageContainer from '../../common/CommonPageContainer';

interface QuickLink {
  id: number;
  url: string;
  text: string;
}

const LinkElement = ({ url, text, id }: QuickLink): JSX.Element => (
  <a href={url} key={id} style={{ textDecoration: 'none', padding: '8px 8px', color: 'black' }}>{text}</a>
);

export default function ReviewAndCancelTransfers() {
  const quickLinks: QuickLink[] = [
    {
      id: 1,
      text: 'Set up a transfer',
      url: './',
    },
    {
      id: 2,
      text: 'Edit your "Postdated Transfer" alerts',
      url: './',
    },
  ];
  return (
    <div className="page-content">
      <CommonPageContainer title="Review & Cancel Transfers">
        <Row>
          <Col md={9}>
            <h6 style={{ marginTop: '20px 0px' }}>Overview</h6>
            <ul>
              <li>
                To delete an upcoming transfer,
                check the box next to it, then select &ldquo;Delete Transfer&ldquo;.
              </li>
              <li>
                For a recurring transfer,
                this will delete the next transfer and all remaining
                transfers in the series.
              </li>
            </ul>
          </Col>
          <Col md={3}>
            <Row style={{ border: 'solid #AAAAAA 1px', justifyItems: 'space-between', margin: '0px 0px 20px 0px' }}>
              <h6 style={{ paddingTop: '10px' }}>You can also:</h6>
              {quickLinks.map((q) => (LinkElement(q)))}
            </Row>
          </Col>
        </Row>

        <hr style={
        {
          height: '2px',
          color: 'C4C4C4',
          margin: '0px',
        }
      }
        />
        <Table className="table table-borderless">
          <thead style={
          {
            height: '40px',
            background: '#EEEEEE',
            margin: '0px',
          }
        }
          >
            <tr>
              <th scope="col" style={{ width: '20%' }}>FROM</th>
              <th scope="col" style={{ width: '20%' }}>TO</th>
              <th scope="col" style={{ width: '15%' }}>AMOUNT</th>
              <th scope="col" style={{ width: '15%' }}>DATE</th>
              <th scope="col" style={{ width: '30%' }}>DETAILS</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td colSpan={5}>
                <Row style={{ justifyContent: 'center', margin: '20px 0' }}>
                  You do not have any upcoming transfers.
                </Row>
              </td>
            </tr>
          </tbody>
        </Table>
        <hr style={
        {
          height: '2px',
          color: 'C4C4C4',
          margin: '0px',
        }
      }
        />
      </CommonPageContainer>
    </div>
  );
}
