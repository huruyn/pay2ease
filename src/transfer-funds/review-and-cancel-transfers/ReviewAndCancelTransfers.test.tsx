import React from 'react';
import ReviewAndCancelTransfers from './ReviewAndCancelTransfers';

const ReactTestRenderer = require('react-test-renderer');

describe('ReviewAndCancelTransfers Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(
      <ReviewAndCancelTransfers />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
