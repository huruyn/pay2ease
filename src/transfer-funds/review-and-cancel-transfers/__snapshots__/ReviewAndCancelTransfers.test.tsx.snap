// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewAndCancelTransfers Component matches the snapshot 1`] = `
<div
  className="page-content"
>
  <div
    className="d-flex justify-content-between align-items-center"
  >
    <h3>
      Review & Cancel Transfers
    </h3>
  </div>
  <div
    className="content-wrapper"
  >
    <div
      className="row"
    >
      <div
        className="col-md-9"
      >
        <h6
          style={
            Object {
              "marginTop": "20px 0px",
            }
          }
        >
          Overview
        </h6>
        <ul>
          <li>
            To delete an upcoming transfer, check the box next to it, then select “Delete Transfer“.
          </li>
          <li>
            For a recurring transfer, this will delete the next transfer and all remaining transfers in the series.
          </li>
        </ul>
      </div>
      <div
        className="col-md-3"
      >
        <div
          className="row"
          style={
            Object {
              "border": "solid #AAAAAA 1px",
              "justifyItems": "space-between",
              "margin": "0px 0px 20px 0px",
            }
          }
        >
          <h6
            style={
              Object {
                "paddingTop": "10px",
              }
            }
          >
            You can also:
          </h6>
          <a
            href="./"
            style={
              Object {
                "color": "black",
                "padding": "8px 8px",
                "textDecoration": "none",
              }
            }
          >
            Set up a transfer
          </a>
          <a
            href="./"
            style={
              Object {
                "color": "black",
                "padding": "8px 8px",
                "textDecoration": "none",
              }
            }
          >
            Edit your "Postdated Transfer" alerts
          </a>
        </div>
      </div>
    </div>
    <hr
      style={
        Object {
          "color": "C4C4C4",
          "height": "2px",
          "margin": "0px",
        }
      }
    />
    <table
      className="table table-borderless table"
    >
      <thead
        style={
          Object {
            "background": "#EEEEEE",
            "height": "40px",
            "margin": "0px",
          }
        }
      >
        <tr>
          <th
            scope="col"
            style={
              Object {
                "width": "20%",
              }
            }
          >
            FROM
          </th>
          <th
            scope="col"
            style={
              Object {
                "width": "20%",
              }
            }
          >
            TO
          </th>
          <th
            scope="col"
            style={
              Object {
                "width": "15%",
              }
            }
          >
            AMOUNT
          </th>
          <th
            scope="col"
            style={
              Object {
                "width": "15%",
              }
            }
          >
            DATE
          </th>
          <th
            scope="col"
            style={
              Object {
                "width": "30%",
              }
            }
          >
            DETAILS
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td
            colSpan={5}
          >
            <div
              className="row"
              style={
                Object {
                  "justifyContent": "center",
                  "margin": "20px 0",
                }
              }
            >
              You do not have any upcoming transfers.
            </div>
          </td>
        </tr>
      </tbody>
    </table>
    <hr
      style={
        Object {
          "color": "C4C4C4",
          "height": "2px",
          "margin": "0px",
        }
      }
    />
  </div>
  <div
    className="footer-content-wrapper"
  >
    <div
      className="quick-links container"
    >
      <div
        className="row"
      >
        <h3
          className="col-sm-12"
        >
          Quick Links
        </h3>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-2"
        >
          <ul>
            <li>
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li>
              Please Review Our Legal Stuff by visiting our
               
              <a
                className="text-decoration-none"
                href="/legal"
                onClick={[Function]}
              >
                Privacy Policy | TOU | EULA | CardHolder T&C Pages
              </a>
            </li>
          </ul>
        </div>
        <div
          className="col-sm-2"
        >
          8465 W Sahara Ave. Las Vegas, NEVADA
          <ul>
            <li>
              <a
                href="tel:+17026231876"
              >
                1.702.623.1876
              </a>
            </li>
            <li>
              <a
                href="mailto:contact@zippy.cash"
              >
                contact@zippy.cash
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12 col-sm-12 col-xs-12 text-left"
        >
          © 2021 Zippy Cash LLC  All rights reserved.
          <br />
          <br />
          The Program Manager for the Zippy Cash® Prepaid Visa Debit Card is PayQuicker, LLC
          <br />
          Zippy Cash® is a registered trademark of Zippy Cash, LLC
          <br />
          The Zippy Cash Prepaid Visa Debit Card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Visa U.S.A. Inc. “Metropolitan Commercial Bank” and “Metropolitan” are registered trademarks of Metropolitan Commercial Bank ©2014. BY USING THIS CARD YOU AGREE TO THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. All trademarks and service marks are the property of their respective owners.
          <br />
          In compliance with the USA PATRIOT Act* and other laws, anyone who is a US resident over 18 with a valid Social Security number and a verifiable physical US address (no PO boxes or APO addresses) can apply for an Zippy Cash Card. This information must be provided, along with other information pertinent to receiving approval, when applying for a Zippy Cash Card. Currently, we do not issue the Zippy Cash Card to residents of Puerto Rico, Guam, the Virgin Islands, American Samoa, or other US territories.
          <br />
          *Important information for opening a card account: To help the federal government fight the funding of terrorism and money laundering activities, the USA PATRIOT Act requires all financial institutions and their third parties to obtain, verify and record information that identifies each person who opens a card account. What this means for you: When you open a card account, we will ask for your name, address, date of birth, social security number or country identification number, and other information that will allows us to identify you. We may also request a copy of your driver’s license or other documentation bearing your photo as verification of your identity.
          <br />
          Apple and the Apple logo are trademarks of Apple Inc., registered in the U.S. and other countries. App Store is a service mark of Apple Inc. Google Play is a trademark of Google Inc. Data rates may apply. Samsung Pay is a trademark of Samsung Inc.
        </div>
      </div>
    </div>
    <footer
      className="container mb-3"
    >
      <div
        className="d-none d-md-block"
      >
        <div
          className="d-flex justify-content-between border-bottom"
        />
        <div
          className="d-flex justify-content-between border-top"
        >
          <div
            className=""
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
          <div
            className="copyright mt-1"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
      <div
        className="d-block d-md-none"
      >
        <div
          className="d-flex border-bottom"
        >
          <div
            className="d-flex flex-column me-2"
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
        </div>
        <div
          className="d-flex justify-content-end border-top"
        >
          <div
            className="copyright mt-2 text-end small"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
